WordPressAuthenticator/Signin/Login2FAViewController.swift

```diff
@@ -97,6 +97,7 @@ class Login2FAViewController: LoginViewController, NUXKeyboardResponder, UITextF
         sendCodeButton.setTitle(NSLocalizedString("Text me a code instead", comment: "Button title"),
                                 for: .normal)
         sendCodeButton.titleLabel?.numberOfLines = 0
+        sendCodeButton.applyTextButtonStyle()
```

Comment:
I don't feel like this call belongs in a method named `localizeControls`, which is all around localising and setting text. We could just create a new `configureTextButton` method to include this call?